setup.cfg

```diff
@@ -4,12 +4,8 @@ commit = True
 tag = True
 
 [bumpversion:file:setup.py]
-search = version='{current_version}'
-replace = version='{new_version}'
 
 [bumpversion:file:pepys_import/__init__.py]
```

Comment:
@robintw  - just checking.  We still need the text at line 6, just to indicate to BumpVersion that it should include `setup.py`, right?  I'm asking because we don't provide such an identifier for the third file that gets "bumped", `setup.cfg`.

Reply:
Yeah we do. `setup.cfg` is acting as the bumpversion config file here, and bumpversion automatically updates its own config file.